imod/util/time.py

```diff
@@ -189,7 +189,7 @@ def _compose_timestring(
     types of available time objects.
     """
     if time == "steady-state":
-        return time
+        return str(time)
```

Comment:
Please add comment that this is to avoid a mypy error. 

For the common observer, it makes sense that ``"steady-state"`` is always a string, so this string conversion appears trivial.

Reply:
done